src/test/cypress/e2e/Logout.cy.ts

```diff
@@ -1,5 +1,6 @@
-import { Course } from '../../../main/webapp/app/entities/course.model';
-import { ModelingExercise } from '../../../main/webapp/app/entities/modeling-exercise.model';
+import { Course } from 'app/entities/course.model';
+import { ModelingExercise } from 'app/entities/modeling-exercise.model';
+
```

Comment:
are these empty lines on purpose?

Reply:
Yes these empty lines are on purpose to separate the following types of imports:
1. Imports from external dependencies (e.g. `dayjs`)
2. Imports from the client code
3. Imports from within the e2e environment